src/foam/nanos/auth/Group.js

```diff
@@ -185,7 +185,7 @@ if ( session != null ) {
       //populate AppConfig url with request's RootUrl
       HttpServletRequest req = x.get(HttpServletRequest.class);
 
-      if ( config.getEnableForceHttps() ) {
+      if ( config.getForceHttps() ) {
```

Comment:
Just a style suggestion.  Usually for boolean properties isWhatever() is oft recommended over getWhatever() for readbility purposes.  Assuming the is-verb methods are generated by foam2:
```suggestion
      if ( config.isForceHttps() ) {
```

Reply:
Java generation - we generate getFoo, on javascript you have access to the variable so just 'foo'
so for javascipt - 'foo' is fine, and I would expect getFoo from java.